catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryResponseImpl.java

```diff
@@ -50,6 +50,8 @@
 
     List<Result> resultList = null;
 
+    private long timeout;
```

Comment:
I would recommend setting a default value in the class to avoid initialization race conditions.